### PR TITLE
Fix the earliest last modified age of translog issue.

### PR DESCRIPTION
**Pull request by howardhuanghua**
Currently translog's `earliest_last_modified_age` field is always 0 in `_nodes/stats` response.
```
GET _nodes/stats?pretty
Response:
....
 "translog" : {
      "operations" : 87590,
      "size_in_bytes" : 181674457,
      "uncommitted_operations" : 87590,
      "uncommitted_size_in_bytes" : 181674457,
      "earliest_last_modified_age" : 0
    },
....
```

### CODE REVIEW

**Review by dnhatn — COMMENTED** [review]
Thanks @howardhuanghua. I prefer fixing the issue when [adding](https://github.com/elastic/elasticsearch/blob/4c05d25ec7ddd4049bba5eed9b3cb51f50f0fee7/server/src/main/java/org/elasticsearch/index/translog/TranslogStats.java#L91-L92) a translog stats instead.

**Comment by howardhuanghua** [comment]
Thanks @dnhatn , you mean add a extra check to `this.earliestLastModifiedAge`, if it's `0L` then set `this.earliestLastModifiedAge = translogStats.earliestLastModifiedAge` directly? Why would we just give a max long initial value for `this.earliestLastModifiedAge`? It seems much more simple.

**Comment by dnhatn** [comment]
Yes, but 0L is more appropriate than Long.MAX_VALUE for the timestamp of an empty translog.

**Comment by howardhuanghua** [comment]
Hi @dnhatn, I have updated the change, please help to review again, thanks.

File: server/src/main/java/org/elasticsearch/index/translog/TranslogStats.java [hunk 1]
```diff
@@ -80,8 +80,12 @@ public void add(TranslogStats translogStats) {
         this.translogSizeInBytes += translogStats.translogSizeInBytes;
         this.uncommittedOperations += translogStats.uncommittedOperations;
         this.uncommittedSizeInBytes += translogStats.uncommittedSizeInBytes;
-        this.earliestLastModifiedAge =
-            Math.min(this.earliestLastModifiedAge, translogStats.earliestLastModifiedAge);
+        if (this.earliestLastModifiedAge < 1) {
```
Comment (dnhatn) [inline server/src/main/java/org/elasticsearch/index/translog/TranslogStats.java:83]: Let's use `this.earliestLastModifiedAge == 0`.

**Review by dnhatn — COMMENTED** [review]
Thanks @howardhuanghua. I left a small comment.

File: server/src/main/java/org/elasticsearch/index/translog/TranslogStats.java (hunk 1 above)
Comment (howardhuanghua) [reply server/src/main/java/org/elasticsearch/index/translog/TranslogStats.java]: Updated, thanks.

**Comment by dnhatn** [comment]
@elasticmachine ok to test

**Review by dnhatn — APPROVED** [review]
LGTM. Thanks @howardhuanghua.

**Pull request merged**